keyboards/binepad/bn006/bn006.c

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2022 Binepad (@binpad) */
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include "bn006.h"
+
+led_config_t g_led_config = {
```

Comment:
This could be moved to the info.json, as that supports the rgb matrix config. 

Otherwise: 
```suggestion
#ifdef RGB_MATRIX_ENABLE
led_config_t g_led_config = {
```

Reply:
ohh... didn't know that - docs here https://docs.qmk.fm/#/reference_info_json?id=rgb-lighting only speak of RGB Lighting, not RGB Matrix ... lemme look at the python...

Reply:
... can't find it.  Going old-school. :)